src/test/kotlin/camp/nextstep/edu/immutable/step1/EmployeeTest.kt

```diff
@@ -3,6 +3,7 @@ package camp.nextstep.edu.immutable.step1
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
 
+// NOTE: 실제로는 `src/main/kotlin` 디렉토리에 위치해야 합니다.
```

Comment:
```suggestion
// NOTE: 보통 실무에서는 `src/main/kotlin` 디렉토리에 위치시킵니다. 실습 편의를 위해 테스트 디렉토리에 둡니다.
```
'실제'라는 워딩이 잘 와닿지 않는 것 같은데 설명을 조금 더 구체화해보면 어떨까요! 꼭 제안드린 설명이 아니여도 됩니다.

Reply:
[주석을 조금 더 명확하게 변경한다](https://github.com/next-step/edu-concert-immutable/pull/4/commits/47bb5864fe9298353b06e84d768c18726e1e0314)